core/control-panel/api/src/artifact.rs

```diff
@@ -1,12 +1,12 @@
 use crate::{Sha256HashDTO, TimestampRfc3339, UuidDTO};
 use candid::{CandidType, Deserialize};
 
-#[derive(CandidType, Deserialize, Clone, Debug, Eq, PartialEq)]
+#[derive(CandidType, serde::Serialize, Deserialize, Clone, Debug, Eq, PartialEq)]
```

Comment:
I programatically added `serde::Serialize` to every `derive(CandidType` and `, CandidType` in the `core/*/api` directories.